### DISCUSSION TITLE
Possible to show multiple code blocks as tabs?

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-cli#6196] [Q&A] [answered]
### Description

May I ask if there is a way to show multiple code blocks in the form of `tabs`, like the one shown on this [page](https://d2l.ai/chapter_preliminaries/linear-algebra.html) and the image below?

![image](https://github.com/quarto-dev/quarto-cli/assets/23435810/eb0d63ad-2478-459b-a897-a8dce9dba88d)

**Comment by mcanouil** [comment, marked as answer]
This is documented at <https://quarto.org/docs/output-formats/html-basics.html#tabsets>.

**Reply by hopezh** [reply]
Thanks. I don't know why I overlooked the exact documentation on this.
